doc/user/content/third-party/supported-tools.md

```diff
@@ -0,0 +1,22 @@
+---
+title: "Supported Tools Overview"
+description: "Get details about third-party tool support with Materialize"
+menu:
+  main:
+    parent: 'third-party'
+---
+
+Materialize fully supports the following third-party tools:
```

Comment:
"fully supports" feels a little too aggressive to me. I think something like tiers or alpha-, beta-, production-level would make me feel a little less anxious.